client/scripts/views/pages/discussions/index.ts

```diff
@@ -64,6 +64,8 @@ const getLastSeenDivider = (hasText = true) => {
   );
 };
 
+const onSummaryPage = (p, activeId) => decodeURI(p).endsWith(`/${activeId}/`);
```

Comment:
This won't work on custom domains. This should also be "discussions" page, not Summary. Summary is a view/mode of the discussions page and this should always be true when this `index.ts` file is being rendered, the only difference is when `/discussions/insert_topic_name_here` is added on. Rather than toggling based on lack of topic, if there's a topic being added to the URL, toggle to latest view. This also should *not* need to checked below in multiple places; you use both `summaryViewEnabled` and `summaryView` redundantly.

Reply:
Please check the latest commit.